### PR TITLE
remove inactive reviewers from active accessibilty projects

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/55

### CODE REVIEW

**Comment by cookiecrook** [comment]
Adding @spectranaut as the only reviewer since she's taken over the maintenance of the manual ATTA tests. 

This change will prevent the WPT bot from auto-assigning reviewers who are no longer active, and will keep me from needing to manually unassign them from every PR.

As mentioned in the issue and elsewhere, I confirmed with the removed reviewers (via email) that they should be taken off the auto-assign yml lists, as long as they remained assignable WPT reviewers.

**Comment by cookiecrook** [comment]
Like that! ☝️⬆

**Comment by halindrome** [comment]
I don't mind if you remove me as an active reviewer... if I even am considered one these days.

File: accname/META.yml [hunk 1]
```diff
@@ -3,5 +3,3 @@ suggested_reviewers:
   - cookiecrook
   - spectranaut
   - jnurthen
```
Comment (jnurthen) [inline accname/META.yml:5]:
```suggestion
  - jnurthen
  - melsumner
```

**Comment by cookiecrook** [comment]
> I don't mind if you remove me as an active reviewer... if I even am considered one these days.

As mentioned in the email threads, you'll still be a reviewer... The bot just won't auto-add you as a reviewer to every PR.

**Review by spectranaut — COMMENTED** [review]
seems fine to me, Joanie said she will review my PRs if I ask her :)

**Review by spectranaut — APPROVED** [review]

**Pull request merged**